runtime_config_linux.go

```diff
@@ -21,6 +21,9 @@ type LinuxRuntime struct {
 	// Resources contain cgroup information for handling resource constraints
 	// for the container
 	Resources Resources `json:"resources"`
+	// CgroupsPath specifies the path to cgroups that are created and/or joined by the container.
+	// The path is expected to be relative to the cgroups mountpoint.
+	CgroupsPath string `json:"cgroupsPath"`
```

Comment:
On Tue, Sep 08, 2015 at 03:23:20PM -0700, Daniel, Dao Quang Minh wrote:
> I think that clarifying the order of resources modification is good
> enough i.e, if cgroups path is specified then we join the paths,
> otherwise we create new paths, and then apply the resources limit.

See also comments in #155 and #158, where we're having the same
discussion about namespaces.